newrelic-integration-e2e/internal/runtime/runner.go

```diff
@@ -112,7 +113,16 @@ func (r *Runner) executeOSCommands(statements []string, scenarioTag string) erro
 		cmd.Env = os.Environ()
 		cmd.Env = append(cmd.Env, "SCENARIO_TAG="+scenarioTag)
 		combinedOutput, err := cmd.CombinedOutput()
-		r.logger.Debugf("stdout: %q", combinedOutput)
+
+		if r.spec.PlainLogs {
+			r.logger.WithField("command", stmt).Info(combinedOutput)
+		} else {
+			logWriter := r.logger.Writer()
```

Comment:
After a second thought, I don't think this method does what I thought it did. I'll need to change this.

Reply:
Refactored in https://github.com/newrelic/newrelic-integration-e2e-action/pull/14/commits/41b6412168d17dc1c6f1628e280ba40c3b6f2329